homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/games.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="./css/gamesStyle.css">
+        <link rel="stylesheet" type="text/css" href="./css/main.css">
+        <title>Games</title>
+    </head>
+    <body>
+        <header class="header">
+            <img class="header-logo" src="./Img/logo.svg" width="55px" height="57.323px">
+            <nav class="nav-menu">
+                <ul>
+                    <li>
+                        <a href="./games.html">Games</a>
+                    </li>
+                    <li>
+                        <a href="./index.html">About</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main__section">
+            <h1 class="main-header">Study project "Games"</h1>
+            <div class="main-cards-container">
+                <div class="card">
+                    <div class="main-card-header">
```

Comment:
If you have several simular elements - it's a list. So, it is a list of cards. Please update the structure